precise/iaf_psc_alpha_ps.h

```diff
@@ -0,0 +1,548 @@
+/*
+ *  iaf_psc_alpha_ps.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef IAF_PSC_ALPHA_PS_H
+#define IAF_PSC_ALPHA_PS_H
+
+// C++ includes:
+#include <vector>
+
+// Generated includes:
+#include "config.h"
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+// Includes from precise:
+#include "slice_ring_buffer.h"
+
+namespace nest
+{
+
+/** @BeginDocumentation
+Name: iaf_psc_alpha_ps - Leaky integrate-and-fire neuron
+with alpha-shape postsynaptic currents (one timescale for excitatory and
```

Comment:
I would remove the text in (), since this model should become the default. Also, in this model we rather have *different* timescales, as it is written now one could think there is just a single for ex *and* in.